assets/src/js/pdf_icon.js

```diff
@@ -13,5 +13,7 @@ export const setupPDFIcon = () => {
     }
 
     link.classList.add('pdf-link');
+    const color = window.getComputedStyle(link).getPropertyValue('color');
```

Comment:
Pretty sure we don't want to be calling `getComputedStyle` during page load since it causes reflow.